### ISSUE TITLE
Add contributors highlight + show "humans" of ml5

### ISSUE THREAD

**Issue by joeyklee** [ml5js/ml5-website-archive#4]
We've got so many wonderful contributors - it would be nice to show that there's humans behind the project: https://github.com/ml5js/ml5-library/graphs/contributors

<img width="1376" alt="Screen Shot 2019-04-19 at 15 12 24" src="https://user-images.githubusercontent.com/3622055/56439870-a0546980-62b5-11e9-8ce7-cc6360e34ac5.png">

maybe on an about page or the home page?

**Comment by shiffman** [comment]
Yes, I agree! I'll point out, however, that GitHub automated metrics aren't necessarily an accurate reflection of the community of contributors so a more curated approach might be best. p5 does a nice job of this here

https://github.com/processing/p5.js#contributors
https://github.com/all-contributors/all-contributors

**Comment by joeyklee** [comment]
@shiffman - Yes 100%. Github metrics are not a great indicator for contributions. The p5 contributors page is awesome!

**Comment by joeyklee** [comment]
We can add the contributors from: https://github.com/ml5js/ml5-library#contributors to the website under "about" ?

we can `fetch` the github API to get the latest when visitors come to the site? @wenqili

**Comment by bomanimc** [comment]
I really like the idea of adding the contributors as a featured section of our website. 

I've felt that I don't have a great understanding of the system for our GitHub version on ml5-library, which seems to show a repeated list of folks and may not be capturing everyone. Would love to pick up discussion here about how we can improve or expound up systems that help to celebrate all of the awesome contributors to the project, especially more broadly than GitHub-using contributors. cc @c-dacanay for ideas about this too!
